vue-components/src/components/LookupMenu.vue

```diff
@@ -15,6 +15,9 @@
 				{{ menuItem.description }}
 			</div>
 		</div>
+		<div v-if="menuItems.length === 0" class="wikit-LookupMenu__no-results">
```

Comment:
So from this I gather that the default state of the component shows items (even before an initial lookup started), right? Otherwise it would show this slot as the message initially.

Reply:
Mh, the menu should probably only show after we received the first response to our auto-complete query.

Reply:
(Though that's something that has to be implemented in the Lookup component, not in the LookupMenu)

Reply:
Mh, thinking more about this: the prop `menuItems` on `Lookup` should probably default to `null` instead of an empty list to be able to differentiate between no auto-complete response yet and an empty response.

Reply:
As far as I can see this problem goes away as soon as we have way to indicate the "loading" state. What do you think about building that into the Lookup in a forward-compatible way instead of modelling this initial lookup state?

Reply:
I didn't see anything about a loading state in the design, but if we were to add such a loading state, that could resolve this issue as well. Though we have to be careful about race conditions when implementing it.

That being said, I consider all this to be material for incremental improvements. The current code would be fine for me for the initial version of the component.

Not sure what @itamargiv thinks about this :)